WcaOnRails/db/seeds/development/competitions.seeds.rb

```diff
@@ -70,8 +70,8 @@ def random_wca_value
             value3: random_wca_value,
             value4: random_wca_value,
             value5: random_wca_value,
-            regionalSingleRecord: i == 0 ? "WR" : "",
-            regionalAverageRecord: i == 0 ? "WR" : "",
+            regionalSingleRecord: j == 0 ? "WR" : "",
+            regionalAverageRecord: j == 0 ? "WR" : "",
```

Comment:
Why would you expect them to not be allowed here? There was something about using double quotes inside of interpolation inside of double quotes, but this is not related to that.